Detectors/FIT/FT0/base/include/FT0Base/Geometry.h

```diff
@@ -37,19 +43,38 @@ class Geometry
 
   ///
   TVector3 centerMCP(int imcp) { return mMCP[imcp]; }
+  TVector3 tiltMCP(int imcp) { return mAngels[imcp]; }
 
-  static constexpr int Nchannels = 229;                     // number od channels
+  static constexpr int Nchannels = 229;                     // number of LUT channels
+  static constexpr int Nsensors = 208;                      // number of channels
   static constexpr int NCellsA = 24;                        // number of radiatiors on A side
   static constexpr int NCellsC = 28;                        // number of radiatiors on C side
   static constexpr float ZdetA = 335.5;                     // number of radiatiors on A side
   static constexpr float ZdetC = 82;                        // number of radiatiors on C side
   static constexpr float ChannelWidth = 13.02;              // channel width in ps
   static constexpr float ChannelWidthInverse = 0.076804916; // channel width in ps inverse
+  static constexpr o2::detectors::DetID::ID getDetID() { return o2::detectors::DetID::FT0; }
+  void setAsideModules();
+  void setCsideModules();
+  const char* getSymbolicName(int index) const
+  {
+    /// return symbolic name of sensor
+    LOG(INFO) << o2::base::GeometryManager::getSymbolicName(getDetID(), index);
+    return o2::base::GeometryManager::getSymbolicName(getDetID(), index);
+  }
+  TGeoPNEntry* getPNEntry(int index) const
+  {
+    /// Get a pointer to the TGeoPNEntry of a chip identified by 'index'
+    /// Returns NULL in case of invalid index, missing TGeoManager or invalid symbolic name
+    return o2::base::GeometryManager::getPNEntry(getDetID(), index);
+  }
 
  private:
   TVector3 mMCP[52];
+  TVector3 mAngels[28];
+  static std::string sSensorName; ///< Sensor name
```

Comment:
I don't see where this `sSensorName` is used. Btw, I guess you meant `mAngles`, not `mAngels`